src/adaptive_graph_of_thoughts/domain/models/graph_elements.py

```
@@ -14,233 +15,184 @@
-    # Hyperedge specific (virtual edge type if representing hyperedges in a DiGraph)
     HYPEREDGE_COMPONENT = "hyperedge_component"
-
     OTHER = "other"
 
 
-# --- Metadata Sub-Models (aligning with P1.12) ---
-class FalsificationCriteria(BaseModel):  # P1.16
+# --- Metadata models ---
+class FalsificationCriteria(BaseModel):
     description: str
     testable_conditions: List[str] = []
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)